.github/workflows/test.yml

```diff
@@ -8,12 +8,26 @@ on:
             - "releases/**"
         tags:
             - "*"
+        paths-ignore:
+            - "**.md"
+            - "examples/**"
+            - "**/doc/**"
+            - "data-processing-lib/doc/**"
```

Comment:
could you please add `kfp/doc/**`

Reply:
**/doc/** covers this i believe

Reply:
if this is right, why do you need `data-processing-lib/doc/**` ?